Jenkinsfile_CNP

```diff
@@ -9,6 +9,21 @@ properties(
 def product = "ccd"
 def app = "case-management-web"
 
+List<LinkedHashMap<String, Object>> secrets = [
+    secret('caseworkerAutoTestEmail', 'CCD_CASEWORKER_AUTOTEST_EMAIL'),
+    secret('caseworkerAutoTestPassword', 'CCD_CASEWORKER_AUTOTEST_PASSWORD'),
+    secret('testFrontEndUrl', 'TEST_FRONTEND_URL'),
```

Comment:
I don't think this one is required. I believe Jenkins already provides a `TEST_URL` environment variable which points to the instance of the app which was just deployed.